lib/tapioca/compilers/dsl/state_machines.rb

```diff
@@ -15,10 +15,12 @@
 module Tapioca
   module Compilers
     module Dsl
-      # `RbiGenerator::StateMachines` generates RBI files for classes that setup a `state_machine`
-      # (see https://github.com/state-machines/state_machines). The generator also processes the extra
-      # methods generated by [StateMachines Active Record](https://github.com/state-machines/state_machines-activerecord)
-      # and [StateMachines ActiveModel](https://github.com/state-machines/state_machines-activemodel) integrations.
+      # `Tapioca::Compilers::Dsl::StateMachines` generates RBI files for classes that setup a
+      # [`state_machine`](https://github.com/state-machines/state_machines). The generator also
+      # processes the extra methods generated by
+      # [StateMachines Active Record](https://github.com/state-machines/state_machines-activerecord)
+      # and [StateMachines ActiveModel](https://github.com/state-machines/state_machines-activemodel)
```

Comment:
```suggestion
      # and [StateMachines Active Model](https://github.com/state-machines/state_machines-activemodel)
```

In https://guides.rubyonrails.org/active_model_basics.html they use a space like in `Active Record`.

Reply:
Good catch!